source/Engine.cpp

```diff
@@ -845,6 +846,19 @@ void Engine::Step(bool isActive)
 				int heat = round(100. * target->Heat());
 				info.SetString("target heat", to_string(heat) + "%");
 			}
+			// Actual maneuver information requires a scrutable
+			// target that is within the maneuver scanner range.
+			if ((targetRange <= maneuverScanRange && !target->Attributes().Get("inscrutable"))
```

Comment:
I think we should put this scrutability in a boolean since we're checking for it twice in a row

Reply:
This still needs to be done, actually @Zitchas 
You need to add a line that does this: 
`bool scrutable = target->Attributes().Get("inscrutable")`
It ofc needs to be before we use it here and in the if right before this
And then use scrutable here, and in the previous if.

Reply:
This has now been implemented. Thanks!